dbms/src/Interpreters/NullAwareSemiJoinHelper.h

```diff
@@ -110,9 +110,13 @@ class NASemiJoinResult
     bool step_end;
     NASemiJoinResultType result;
 
-    /// Iterating position of null rows.
+    size_t pace;
+    /// Position of null rows.
     size_t null_rows_pos;
-    Join::RowRefList * null_rows_it;
+    /// Position of MutableColumns vector for a certain null rows.
+    size_t columns_vec_pos;
+    /// Index for a certain MutableColumns.
+    size_t columns_index;
```

Comment:
maybe rename to `pos_in_null_rows/pos_in_columns_vector/pos_in_columns`?

Reply:
Nice name